XenAdmin/Core/Updates.cs

```diff
@@ -459,7 +465,7 @@ private static bool IsSuitableForXenCenterAlert(XenCenterVersion toUse, Version
             return alerts;
         }
 
-        private static bool PatchCanBeInstalledOnHost(XenServerPatch serverPatch, Host host)
+        private static bool PatchCanBeInstalledOnHost(XenServerPatch serverPatch, Host host, XenServerVersion version)

```

Comment:
We should rename the "version" parameter to something like patchApplicableVersion.